Geometry/TrackerNumberingBuilder/src/TrackerShapeToBounds.cc

```diff
@@ -29,25 +29,25 @@
   3. combine the global rotation from DDD with the rotation defined in 2.   
   */
 
-Bounds* TrackerShapeToBounds::buildBounds(const DDSolidShape& _shape, const std::vector<double>& _par) const {
+Bounds* TrackerShapeToBounds::buildBounds(const cms::DDSolidShape& _shape, const std::vector<double>& _par) const {
   switch (_shape) {
-    case DDSolidShape::ddbox:
+    case cms::DDSolidShape::ddbox:
       return buildBox(_par);
       break;
-    case DDSolidShape::ddtrap:
+    case cms::DDSolidShape::ddtrap:
       return buildTrap(_par);
       break;
-    case DDSolidShape::ddtubs:
+    case cms::DDSolidShape::ddtubs:
       return buildOpen(_par);
       break;
-    case DDSolidShape::ddpolycone_rrz:
+    case cms::DDSolidShape::ddpolycone:
```

Comment:
@adewit @ianna ok, is this supposed to handle the old ddpolycone_rrz in a transparent way? Is it mapped into ddpolycone by DD4hep?

Reply:
This code appears to do more than it actually does. It should be simplified for clarity.
```
case cms::DDSolidShape::ddtubs:
case cms::DDSolidShape::ddpolycone:
case cms::DDSolidShape::ddsubtraction:
  return buildOpen();
  break;
```
Note that `buildOpen()` should be changed to take no parameter. If you check the definition of the method, it doesn't use its parameter.